languages/python/example.py

```diff
@@ -14,10 +14,10 @@
 
 # Add some logging & set the org id
 logging.basicConfig(level=logging.DEBUG)
-organization_id = "org_id_here"
+organization_id = os.getenv("BWS_ORG_ID")
```

Comment:
Minor thing, but could we change these env variables to be the same we use on the other languages? That way it's easier to test them all at the same time. We could also use env variables for the api and identity URLs.

These are the names we're using with the rest of integrations:
https://github.com/bitwarden/sdk/blob/679673096254e108fa0d71f65bf8fa28ba9679f5/languages/go/example/example.go#L13-L19